app/views/goodsRecord/SingleRecordView.scala.html

```diff
@@ -49,18 +49,36 @@ <h3 class="govuk-notification-banner__heading">
         )
     }
 
-
     <h1 class="govuk-heading-l">@messages("singleRecord.h1")</h1>
     <h2 class="govuk-heading-m">@messages("singleRecord.h2")</h2>
 
     <p class="govuk-body">@messages("singleRecord.status")
-
-        <td class="govuk-table__cell">
-            <strong class="govuk-tag govuk-tag--red">
-                @messages(declarableStatus.messageKey)
-            </strong>
-        </td>
+        @declarableStatus match {
+            case DeclarableStatus.ImmiReady => {
+                <strong class="govuk-tag govuk-tag--green">
+                    @messages(declarableStatus.messageKey)
+                </strong>
+            }
+            case DeclarableStatus.NotReadyForImmi => {
+                <strong class="govuk-tag govuk-tag--orange">
+                    @messages(declarableStatus.messageKey)
+                </strong>
+            }
+            case DeclarableStatus.NotReadyForUse => {
+                <strong class="govuk-tag govuk-tag--red">
+                    @messages(declarableStatus.messageKey)
+                </strong>
+            }
+            case _ => {}
+        }
```

Comment:
A further refactor, if you wanted is, for the above you could also add the tag class to the DeclarableStatus so we could do
  
```
  sealed trait DeclarableStatus {
  val messageKey: String
  val paragraphKey: String
  val paragraphTagColour: String
}
```

```
  case object NotReadyForImmi extends DeclarableStatus {
    val messageKey   = "declarableStatus.notReadyForImmi"
    val paragraphKey = "declarableStatus.notReadyForImmi.paragraph"
    val paragraphTagColour = "govuk-tag--orange"
  }

```
Which means in our code we can just do this, without the case match as well:

```
<p class="govuk-body">@messages("singleRecord.status")
                <strong class=s"govuk-tag ${declarableStatus.paragraphTagColour}">
                    @messages(declarableStatus.messageKey)
                </strong>
        </p>
```
The reason we can get rid of the match, alongside this change, is because it looks like DeclarableStatus is not optional, so we will always have it displaying on the page.

Reply:
You might have to mess with the twirl for the ${declarableStatus.paragraphTagColour} bit, as it might need a @, I know twirl can be a bit weird with the syntax